### DISCUSSION TITLE
Can't change account e-mail

### DISCUSSION THREAD

**Discussion by ZedroPilves** [vercel/community#3813] [Account Issue] [answered]
### Question

I tried to create an account with mi original e-mail and it got blocked, i created with other email and now i can't change it bc it says that the account is blocked

**Comment by amyegan** [comment, marked as answer]
Hey @Zedro-Pilves. If you have not already, please email [registration@vercel.com](mailto:registration@vercel.com) and include the email addresses and details of the project you want to deploy. Due to the sensitive nature of account information, we cannot safely discuss it in this public forum.

**Comment by oaojodrepmagalhaes** [comment]
nao consigo logar
